modules/oe_content_event/README.md

```diff
@@ -57,3 +57,22 @@ This module provides integration with the OpenEuropa Link List component in the
 plugin that allows to filter Event link lists based on their start and end dates.
 To ensure proper cacheability, this plugin depends on the `openeuropa/oe_time_caching` component so make sure you
 require it in your composer.json file.
+
+## Event daterange fields replacement to daterange_timezone from version 3.x
+
+As of 3.x the field type of the following fields ahs been changed to daterange_timezone in order to support timezones:
+  - oe_event_dates
+  - oe_event_online_dates
+  - oe_event_registration_dates
+
+This change has been carried out via a helper class called `EventDateRangeFieldTypeChanger` where we directly execute
+the necessary MySQL commands to ensure seamless performant change in the background by maintaining the same machine-names.
```

Comment:
```suggestion
the necessary MySQL queries to ensure seamless performant change in the background by maintaining the same machine names.
```

Reply:
he necessary database queries to ensure seamless performant change in the background by maintaining the same machine names.